### PR TITLE
decimal tranche space example

### PR DESCRIPTION
**Pull request by thedavidmeister**
## Motivation

bring pubstrats up to date with decimals

## Solution

A few things ended up happening here

- Convert literals from `1e18` to `1` and proportional
- delete tutorial example that is now in gist https://gist.github.com/thedavidmeister/d1c21bb3a6d6ebedcd44cdc70ce42597
- move unbound strats into WIP folder (untested)
- updated words to use new math words
- tested basic deploy of all strats
- updated subparser addresses for flare
- removed vault ids from bindings
- some minor refactors of the strats to simplify

### CODE REVIEW

File: src/learning/flare/flr-ftso-market-order.rain
Comment (hardyjosh) [inline src/learning/flare/flr-ftso-market-order.rain:1]: @thedavidmeister should we add a one-way switch to this order?

File: src/learning/flare/ftso-spread.rain
Comment (hardyjosh) [inline src/learning/flare/ftso-spread.rain:1]: @thedavidmeister i think this one could do with a one-way switch also

**Review by hardyjosh — CHANGES_REQUESTED** [review]
We should also add a clear disclaimer at the top of each strat that these are examples and people should try to understand for themselves what they are deploying.

File: src/learning/flare/ftso-spread.rain
Comment (thedavidmeister) [reply src/learning/flare/ftso-spread.rain:1]: https://github.com/rainlanguage/rain.dex.pubstrats/issues/48

**Comment by thedavidmeister** [comment]
@hardyjosh https://github.com/rainlanguage/rain.dex.pubstrats/issues/47

**Pull request merged**